cg/cli/deliver/base.py

```diff
@@ -133,6 +133,54 @@ def deliver_ticket(
     )
 
 
+@deliver.command(
+    name="sample",
+    help="Deliver fastq or bam files for a sample to the customer inbox on the HPC "
+    "and start an Rsync job to clinical-delivery. "
+    "NOTE: the dry-run flag will copy files to the customer inbox on Hasta, "
+    "but will not perform the Rsync job.",
+)
+@click.pass_obj
+@click.option(
+    "-c",
+    "--case-id",
+    required=True,
+    help="The case the sample is on.",
+)
+@click.option("-s", "--sample-id", required=True, help="Deliver the files for a specific sample.")
+@click.option(
+    "-d",
+    "--delivery-type",
+    required=True,
+    help="The delivery type to use.",
+    type=click.Choice(choices=["fastq", "bam"]),
+)
+@DRY_RUN
+def deliver_sample_raw_data(
```

Comment:
Should this command be part of this PR? Seems a bit disjoint from the post process errors?